README.md

```diff
@@ -1 +1,102 @@
-# Apache Ignite 3.x
+Apache Ignite 3.x
+===
+
+Apache Ignite 3 is a distributed database for high-performance computing with in-memory speed.
```

Comment:
Copy&past from Ignite 2? :) I believe we should reconsider this statement, because, IMHO, the in-memory is not the main mode and is not the main use-case.